### ISSUE TITLE
Nested @media rules render different from Ruby SASS

### ISSUE THREAD

**Issue by mgreter** [sass/libsass#185]
Hi everybody

Nested @media rules are not rendered the same as with Ruby SASS.

Source SCSS File:
```
@media only screen and (max-width : 100px) {
	.outer { width: 10px; }
	@media only screen and (max-width : 200px) {
		.inner { width: 20px; }
	}
}
```

Libsass output:
```
@media only screen and (max-width: 100px) {
  .outer {
    width: 10px; }
  @media only screen and (max-width: 200px) {
    .inner {
      width: 20px; } } }
```

Ruby SASS output:
```
@media only screen and (max-width: 100px) {
  .outer {
    width: 10px; } }
@media only screen and (max-width: 100px) and (max-width: 200px) {
  .inner {
    width: 20px; } }
```

Although libsass output is valid css3, it's not valid css2.1.
For some more info see: http://stackoverflow.com/a/11747166

Thanks and have a nice day!

**Comment by xzyfer** [comment]
This fixed via #800 and will be in 3.2.

**Comment by msikma** [comment]
I'm very glad this will be fixed with 3.2, this will make it easy to do e.g. retina-specific code inside code blocks for specific responsive sizes. Hoping the next release is finalized soon, any guesses as to when it'll be ready?

**Comment by xzyfer** [comment]
@msikma it's hard to say at the moment. You can https://github.com/sass/libsass/issues/812 to keep up to date on the 3.2 release.
